migrate.sh

```diff
@@ -454,7 +480,7 @@ BEGIN
     DECLARE cur CURSOR FOR
         SELECT table_name, column_name, column_type, extra, data_type
         FROM information_schema.columns
-        WHERE column_key = 'PRI';
+        WHERE column_key = 'PRI' AND table_schema = 'homeassistant';
```

Comment:
You should have something like ${DB_NAME} instead of hardcoded 'homeassistant' here.
Sorry, I should have been more specific in my suggestions.

Reply:
You're right. Should be an easy fix.